backend/graphql/resolvers/testResolvers.ts

```diff
@@ -148,6 +148,21 @@ const testResolvers = {
     ): Promise<TestResponseDTO | null> => {
       return testService.duplicateTest(id);
     },
+    deleteAssessment: async (
+      _req: undefined,
+      { id }: { id: string },
+    ): Promise<string> => {
+      const testToDelete = await testService.getTestById(id);
```

Comment:
sorry for not mentioning this earlier... but do you mind moving this logic into the service directly? (i.e. updating `testService.deleteTest(id)` to do either a soft or hard delete depending on status)

Reply:
this way, you will not need to write a resolver (since `deleteTestById` already exists)